www/dvop06/hw7/css/main.css

```diff
@@ -0,0 +1,36 @@
+* {
+    box-sizing: border-box;
+}
+body {
+    font-family: 'Open Sans', sans-serif;
+    text-align: center;
+}
+#game-field {
+    margin: 10px auto;
+    display: flex;
+    flex-wrap: wrap;
+    width: 95%;
+    justify-content: space-around;
+}
+
+/* dumb way of 'not displaying' the text on card */
+.card {
+    background-color: crimson;
+    color: crimson;
+    width: 17%;
+    height: 125px;
+    margin: 10px 0;
+    border: solid #ddd 3px;
+    border-radius: 5px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
```

Comment:
vlastnost `user-select: none;` -> `var mind = 'blown'`
nebo to muzes resit pres CSS `display: none` ditete tedy napr.:
```css
.card span {
   display: none;
}
.card.revealed span {
   display: initial;
}
```